src/v1/sources/cordial/transform.js

```diff
@@ -0,0 +1,44 @@
+const Message = require('../../../v0/sources/message');
+const { CommonUtils } = require('../../../util/common');
+const { generateUUID, isDefinedAndNotNull } = require('../../../v0/util');
+const { eventsMapping } = require('./config');
+
+const mapping = require('./mapping.json');
+
+const processEvent = (event) => {
+  const message = new Message(`Cordial`);
+  let eventName = event.event?.a || event.event?.action;
+  if (eventName in eventsMapping) {
+    eventName = eventsMapping[eventName];
+  }
+  message.setEventType('track');
+  message.setEventName(eventName);
+  message.setPropertiesV2(event, mapping);
+
+  const externalId = [];
+  // setting up cordial contact_id to externalId
+  if (event.cID) {
+    externalId.push({
+      type: 'cordialContactId',
+      id: event.cID,
+    });
+  }
+  message.context.externalId = externalId;
+
+  if (!isDefinedAndNotNull(message.userId)) {
+    message.anonymousId = generateUUID();
```

Comment:
The length of this UUID is not same length as the anonymousId generated by SDKs can you once verify that?

Reply:
We are using this same function everywhere, so I have taken reference from other sources as well.

Reply:
Ok later we need to investigate fine for now